hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -463,6 +487,7 @@ private synchronized boolean tryExpireCurrentLock(boolean fromShutdownHook) {
         return false;
       case SUCCESS:
         logInfoLockState(RELEASED);
+        recordAuditOperation(AuditOperationState.END, System.currentTimeMillis());
```

Comment:
Is there a need to audit other cases including `UNKNOWN_ERROR` and `ACQUIRED_BY_OTHERS`?

Reply:
generally no, in these situations errors will be thrown and the job will fail. the goal here is to verify correctness - silent errors which we would otherwise not know about. i suppose we can record some error state in the operation though. let's wait until the implementation PR so you can deliver final thoughts.